examples/dump_planes.py

```diff
@@ -29,25 +29,11 @@ def dump(ome_tiff_fn):
     print "image size: %d x %d" % (W, H)
     print "pixel type:", dtype
     for i in xrange(img_count):
```

Comment:
Yes, I thought about removing them as well, but I wanted to keep the printed info.